e2etest/pkg/service/validate.go

```diff
@@ -25,3 +27,19 @@ func ValidateL2(svc *corev1.Service) error {
 	}
 	return nil
 }
+
+func ValidateDesiredLB(svc *corev1.Service) {
+	desiredLbIPs := svc.Annotations["metallb.universe.tf/loadBalancerIPs"]
+	if desiredLbIPs == "" {
+		return
+	}
+	framework.ExpectEqual(desiredLbIPs, strings.Join(GetIngressIPs(svc.Status.LoadBalancer.Ingress), ","))
+}
+
+func GetIngressIPs(ingresses []corev1.LoadBalancerIngress) []string {
```

Comment:
this can be private I think

Reply:
right. as it was before. I don't know why I changed it. thanks!